game.cpp

```diff
@@ -890,19 +890,41 @@ void test_game() //!OCLINT tests may be many
     assert(g.get_env().get_wall_s_side() - wall_short_side < 0.00001 &&
            g.get_env().get_wall_s_side() - wall_short_side > -0.00001);
   }
-
-  //shelter moves with ever tick
+  // The game has 42 shelters
+  {
+    const game g;
+    assert(g.get_shelters().size() == 42);
+  }
+  // There are 42 shelters
+  {
+    const int n_shelters{42};
+    const game g(1600, 3, 0, n_shelters);
+    assert(g.get_shelters().size() == n_shelters);
+  }
+  // All shelters have a different location
```

Comment:
Is this a look into future issue to create shapes on random places?

Reply:
Yes, this is connected to #315 :-)